modules/persistence/src/services/metadata/ToC/index.ts

```diff
@@ -48,15 +48,35 @@ const mergeNode = (node: ToC, tocs: ToCInsertions, currentProject) => {
   if (!associatedProject) return node;
   const branches = Object.keys(associatedProject);
   node.options.versions = branches;
-  node.children = branches.map((branch) => {
-    const child = needsUrlifiedToC ? associatedProject[branch].urlified : associatedProject[branch].original;
-    const options = {
-      ...child.options,
-      version: branch,
-    };
-    child.options = options;
-    return child;
-  });
+  node.options.urls = node.options.urls || {};
+  for (const branch of branches) {
+    node.options.urls[branch] = associatedProject[branch]['urlified']?.url;
+  }
+
+  if (node.options?.project === currentProject) {
```

Comment:
@casthewiz this was to flip the injection - merged node for associated project was originally from umbrella, but should be the index slug ("/")

if merged node is for umbrella project (else), we should delete the slug here and resort to the options.urls as first priority on the front end. (with fallbacks to slug / url as already implemented)

Reply:
results from above change:
![image](https://user-images.githubusercontent.com/13054820/216075288-a32330e0-0a44-41b5-9c16-9c7b987d6cfb.png)

![image](https://user-images.githubusercontent.com/13054820/216075416-fc65e417-229d-44e0-9821-aa6a9144b9d7.png)

Reply:
Makes a lot of sense. Thanks for explaining and showing the results here!